File: GTTS/INL.VettingService.Models/GetPersonsVettingHit_Result.cs
using System;
using System.Collections.Generic;
using System.Text;
using INL.Services.Models;

namespace INL.VettingService.Models 
{
    public class GetPersonsVettingHit_Result : BaseResult, IGetPersonsVettingHit_Result
    {
        public long PersonsVettingID { get; set; }
        public int VettingTypeID { get; set; }
        public int HitResultID { get; set; }
        public string HitResultDetails { get; set; }
        public List<GetPersonsVettingHit_Item> items { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-persons-vetting-hit_result.ts


import { GetPersonsVettingHit_Item } from './get-persons-vetting-hit_item';
import { IGetPersonsVettingHit_Result } from './iget-persons-vetting-hit_result';

export class GetPersonsVettingHit_Result implements IGetPersonsVettingHit_Result {
  
	public PersonsVettingID: number = 0;
	public VettingTypeID: number = 0;
	public HitResultID: number = 0;
	public HitResultDetails: string = "";
	public items?: GetPersonsVettingHit_Item[];
  
	public ErrorMessages: string[] = null;
}


